Update license: year & company name

sculpt0r opened the pull request:
Changes made with https://github.com/ckeditor/ckeditor4/blob/master/dev/license/updatelicense.js with a few manual corrections.

vokiel commented on src/lib/utils.js:
```diff
@@ -34,9 +34,9 @@
+				copyright = "/*" + eol + "This software is covered by CKEditor Commercial License. Usage without proper license is prohibited." + eol + "Copyright (c) 2003-" + date.getFullYear() + ", CKSource Holding sp. z o.o. All rights reserved." + eol + "*/" + eol;
 			else
-				copyright = "/*" + eol + "Copyright (c) 2003-" + date.getFullYear() + ", CKSource - Frederico Knabben. All rights reserved." + eol + "For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license" + eol + "*/" + eol;
+				copyright = "/*" + eol + "Copyright (c) 2003-" + date.getFullYear() + ", CKSource Holding sp. z o.o. All rights reserved." + eol + "For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license" + eol + "*/" + eol;
```
Please add trailing slash to the URL:

```suggestion
				copyright = "/*" + eol + "Copyright (c) 2003-" + date.getFullYear() + ", CKSource Holding sp. z o.o. All rights reserved." + eol + "For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license/" + eol + "*/" + eol;
```

Also along with any other places when it occurs.

vokiel commented on test/_assets/samples/ckeditor-dev-correct/samples/old/api.html:
```diff
@@ -202,8 +202,7 @@ <h1 class="samples">
 		<p id="copy">
-			Copyright &copy; 2003-2012, <a class="samples" href="http://cksource.com/">CKSource</a> - Frederico
-			Knabben. All rights reserved.
+			Copyright &copy; 2003-2022, <a class="samples" href="http://cksource.com/">CKSource</a> Holding sp. z o.o. All rights reserved.
```
Use HTTPS (fix in all samples):

```suggestion
			Copyright &copy; 2003-2022, <a class="samples" href="https://cksource.com/">CKSource</a> Holding sp. z o.o. All rights reserved.
```

vokiel commented on test/_assets/samples/ckeditor-dev-correct/samples/old/api.html:
```diff
@@ -202,8 +202,7 @@ <h1 class="samples">
 			CKEditor - The text editor for the Internet - <a class="samples" href="http://ckeditor.com/">http://ckeditor.com</a>
```
Use HTTPS (fix in all samples):

```suggestion
			CKEditor - The text editor for the Internet - <a class="samples" href="https://ckeditor.com/">https://ckeditor.com/</a>
```

vokiel commented on docs/jsduck.json:
```diff
@@ -1,6 +1,6 @@
 {
 	"--title": "CKBuilder Source Code Documentation",
-	"--footer": "Copyright &copy; 2012-2014, <a href=\"http://cksource.com\" style=\"color:#085585\">CKSource</a> - Frederico Knabben. All rights reserved. | Generated with <a href=\"https://github.com/senchalabs/jsduck\">JSDuck</a>.",
+	"--footer": "Copyright &copy; 2012-2014, <a href=\"http://cksource.com\" style=\"color:#085585\">CKSource</a> Holding sp. z o.o. All rights reserved. | Generated with <a href=\"https://github.com/senchalabs/jsduck\">JSDuck</a>.",
```
Use HTTPS

```suggestion
	"--footer": "Copyright &copy; 2012-2014, <a href=\"https://cksource.com/\" style=\"color:#085585\">CKSource</a> Holding sp. z o.o. All rights reserved. | Generated with <a href=\"https://github.com/senchalabs/jsduck\">JSDuck</a>.",
```

vokiel reviewed (changes requested):
To update:
- Trailing slashes.
- HTTP => HTTPS

sculpt0r commented:
@vokiel ready for another review
- added trailing slashes to the license URLs
- used `https` for all URLs
- also, I found a few more places that require updating the year and company name - so there are also updated

vokiel commented:
@sculpt0r  you showed another places we could fix ;-)

Links to addons repo should have `cke4` inside, the same with the CKE4 builder:

`https://ckeditor.com/addons/skins/all` => `https://ckeditor.com/cke4/addons/skins/all`
`https://ckeditor.com/addon/widget` => `https://ckeditor.com/cke4/addon/widget`
`https://ckeditor.com/builder` => `https://ckeditor.com/cke4/builder`

sculpt0r commented:
@vokiel - ready for another review
- `cke4` inserted

Nice that we can correct little something :)

vokiel reviewed (changes requested):
This might take too much time, because after every change I spot some new link that should be changed...

I would skip rest of them besides links to our docs, e.g:
`https://docs.cksource.com/ckeditor_api/symbols/CKEDITOR.dialog.definition` which points to nowhere (old CKSource domain, docs subdomain) => `https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dialog_definition.html`

There are also newer links to docs (in ckeditor.com domain):
`https://docs.ckeditor.com/#!/api/CKEDITOR.dialog.definition` that are redirected correctly to `https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_dialog_definition.html` but we could change them so no redirect will happen.

sculpt0r commented:
@vokiel - ready for another review
- added missed trailing slashes (for license URLs and ckeditor.com)
- replace old docs URLs with the newest one
- replaced URLs causing redirection with the newest one
- replaced old ckeditor nightly  URLs with the newest one

You're right - there are still more things coming 😅

sculpt0r commented:
@vokiel  - not sure how do you proceed with merging?  The creator or reviewer of the PR should merge? In CKE4 the reviewer is merging accepted PR 🤔

vokiel commented:
We also need someone to create new release. Then deploy new version on our download page, and use it in online builder.

Who's gonna do the build/release?

sculpt0r commented:
@jacekbogdanski is doing releases for CKE4. But I have no idea how things are with builder...

vokiel commented:
I managed to build CKBuilder locally and tested if those changes are applied. All seems to be working :ok\_hand: So tomorrow I'll merge the branch and create the `2.4.3` release.

jacekbogdanski commented:
Thanks guys! We've already discussed with @vokiel that he will handle the whole releasing part. We will have to just bump ckbuilder in https://github.com/ckeditor/ckeditor4-presets/blob/master/build.sh#L48 after that.
